packages/atomic/src/components/common/generated-answer/styles/generated-answer.pcss

```diff
@@ -66,3 +75,31 @@
     @mixin mobile-footer-xs-layout;
   }
 }
+
+[part='generated-answer-footer'] {
+  @container (max-width: 600px) {
```

Comment:
just curious, are hard coded pixel stuff the norm in atomic ? cuz it doesn't sound really responsive to me.
(Seeing myself out !  :running_man: )

Reply:
I agree. In this case it means `37.5rem`.